dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithEuTaxonomyNonFinancials.cy.ts

```diff
@@ -0,0 +1,181 @@
+          );
+        }
+        gammaCapexFirstNonAlignedActivity = gammaCapexNonAlignedActivities[0];
+        gammaCapexFirstNonAlignedActivity.activityName = Activity.Education;
+        assertDefined(gammaCapexFirstNonAlignedActivity.share).relativeShareInPercent = 0;
+
+        gammaCapexTotalAmount = assertDefined(gammaCapex.totalAmount);
+      },
+    );
+  });
+
+  it("Check if the panel fetches, converts and displays data correctly", () => {
+    mountMLDTFrameworkPanelFromFakeFixture(
+      DataTypeEnum.EutaxonomyNonFinancials,
+      eutaxonomyNonFinancialsViewConfiguration,
+      fixturesForTests,
+    ).then(() => {
+      const betaTotalAlignedCapexPercentage = roundNumber(
+        assertDefined(betaCapex.alignedShare?.relativeShareInPercent),
+        2,
+      );
+      const gammaCapexTotalAmountFormattedString = formatAmountWithCurrency({
+        amount: assertDefined(gammaCapexTotalAmount.value),
+        currency: assertDefined(gammaCapexTotalAmount.currency),
+      });
+      const gammaTotalAlignedCapexAbsoluteShareString = formatAmountWithCurrency(
+        assertDefined(gammaCapex.alignedShare?.absoluteShare),
+      );
+      const gammaContributionToClimateChangeMitigation = roundNumber(
+        assertDefined(gammaCapex.substantialContributionToClimateChangeMitigationInPercent),
+        2,
+      );
+
+      getSectionHead("Revenue").click();
+      getCellValueContainer("Total Amount", 0)
+        .invoke("text")
+        .should("match", /^0\s*EUR\s.*/);
+      getSectionHead("Revenue").click();
+
+      getSectionHead("CapEx").click();
+      getSectionHead("Aligned Share").click();
+      getCellValueContainer("Relative Share in Percent", 1)
+        .invoke("text")
+        .should("equal", `${betaTotalAlignedCapexPercentage} %`);
+      getCellValueContainer("Absolute Share", 2)
+        .invoke("text")
+        .should("equal", `${gammaTotalAlignedCapexAbsoluteShareString}`);
+      getCellValueContainer("Substantial Contribution to Climate Change Mitigation In Percent", 2)
+        .invoke("text")
+        .should("equal", `${gammaContributionToClimateChangeMitigation} %`);
+
+      getCellValueContainer("Total Amount", 2)
+        .invoke("text")
+        .should("match", new RegExp(`^${gammaCapexTotalAmountFormattedString}\\s.*$`));
+      getCellValueContainer("Total Amount", 2).click();
+      cy.contains("td", gammaCapexTotalAmountFormattedString).should("exist");
+      cy.contains("td", assertDefined(gammaCapexTotalAmount.quality).toString()).should("exist");
+      cy.contains("td", assertDefined(gammaCapexTotalAmount.comment).toString()).should("exist");
+      cy.get(`span[data-test="Report-Download-${assertDefined(gammaCapexTotalAmount.dataSource).fileName}"]`).should(
+        "exist",
+      );
+      cy.get("body").type("{esc}");
+
+      getCellValueContainer("Aligned Activities", 2).click();
+      cy.get("tr")
```

Comment:
Also for these `tr`s it should be guaranteed that they are from a modal.

Reply:
Ok.

Reply:
Ich habe eine wrapper-Funktion eingeführt, welche man um den Code-Block wrappt, den man auf das Modal anwenden möchte.
Dadurch kann man jetzt Cypress-Code explizit auf modals anwenden.

Reply:
Nice